app/grandchallenge/evaluation/templates/evaluation/evaluation_detail.html

```diff
@@ -205,6 +205,21 @@ <h3 class="card-title">Contact User</h3>
 
             {% include "evaluation/evaluation_incomplete_jobs_detail.html" %}
 
+            {% if object.status != object.SUCCESS or incomplete_jobs %}
```

Comment:
Unnecessary check.

```suggestion
            {% if object.status != object.SUCCESS %}
```

Reply:
Actually this will show up when the jobs are in pending states. The only check and display needs to be on the `object.error_message`.